cvat-ui/src/components/register-page/register-form.tsx

```diff
@@ -34,6 +34,8 @@ export interface RegisterData {
 interface Props {
     fetching: boolean;
     userAgreements: UserAgreement[];
+    predifinedEmail?: string;
+    disableNavigation?: boolean;
```

Comment:
Sounds too abstract
I would propose 

```suggestion
    hideLoginLink?: boolean;
```

or something more specific up to you